manager/controllers/app/fybrik_notebook_read_flow_test.go

```diff
@@ -226,3 +226,145 @@ func TestS3NotebookReadFlow(t *testing.T) {
 	}
 	record.Release()
 }
+
+// Test deny by default policy
+func TestS3ReadFlowDeny(t *testing.T) {
+	t.Log("Started TestS3ReadFlowDeny")
+	if s, ok := os.LookupEnv("VALUES_FILE"); !ok ||
+		(s != "charts/fybrik/notebook-test-readflow.values.yaml" &&
+			s != "charts/fybrik/notebook-test-readflow.tls.values.yaml" &&
+			s != "charts/fybrik/notebook-test-readflow.tls-system-cacerts.yaml") {
+		t.Skip("Only executed for notebook tests")
+	}
+	catalogedAsset, ok := os.LookupEnv("CATALOGED_ASSET")
```

Comment:
Why is it needed? You can take the asset id directly from spec.Data[0].DatasetID